inc/class-statify-settings.php

```diff
@@ -142,6 +150,22 @@ public static function options_snippet() {
 		<?php
 	}
 
+	/**
+	 * Option for the nonce for tracking via JS.
+	 *
+	 * @since 1.7.3
+	 *
+	 * @return void
+	 */
+	public static function options_nonce() {
+		?>
+		<input id="statify-nonce" type="checkbox" name="statify[nonce]" value="1" <?php checked( Statify::$_options['nonce'], 1 ); ?>>
+		(<?php esc_html_e( 'Default', 'statify' ); ?>: <?php esc_html_e( 'Yes', 'statify' ); ?>)
+		<br>
+		<p class="description"><?php esc_html_e( 'Activate this setting if the caching time is longer than the nonce time.', 'statify' ); ?></p>
```

Comment:
Should we assume that the average administrator knows about AJAX behavior and nonce validity?

Maybe add an additional note on the observable misbehavior like _longer than the nonce time and you experience “Forbidden“ errors and visits are not counted._

Reply:
I've updated the description. After the release I would add a detailed hint in our [docs](https://statify.pluginkollektiv.org/documentation/settings/#js-tracking).